test/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/request/UpdateVatSubscriptionSpec.scala

```diff
@@ -17,25 +17,44 @@
 package uk.gov.hmrc.vatsubscription.models.updateVatSubscription.request
 
 import uk.gov.hmrc.vatsubscription.assets.TestUtil
-import play.api.libs.json.Json
+import play.api.libs.json.{JsValue, Json}
+import uk.gov.hmrc.vatsubscription.helpers.{DeclarationTestConstants, DeregistrationInfoTestConstants}
 import uk.gov.hmrc.vatsubscription.helpers.UpdateVatSubscriptionTestConstants._
 import uk.gov.hmrc.vatsubscription.models.{MTDfBExempt, MTDfBMandated}
 
 class UpdateVatSubscriptionSpec extends TestUtil {
 
+  val updateVatSubscriptionLatestDESApi1365JsonMax: JsValue = Json.obj(
+    "messageType" -> messageType,
+    "controlInformation" -> ControlInformation(welshIndicator = false, mandationStatus = Some(MTDfBMandated)),
+    "requestedChange" -> Json.toJson(changeAll)(RequestedChanges.DESApi1365Writes(mockAppConfig)),
+    "contactDetails" -> Json.toJson(updatedPPOB),
+    "returnPeriods" -> Json.toJson(updatedReturnPeriod),
+    "deregistrationInfo" -> DeregistrationInfoTestConstants.deregInfoCeasedTradingDESJson,
+    "declaration" -> DeclarationTestConstants.declarationDESJsonAgent
+  )
+
+  val updateVatSubscriptionLatestDESApi1365JsonMin: JsValue = Json.obj(
+    "messageType" -> messageType,
+    "controlInformation" -> ControlInformation(welshIndicator = false),
+    "requestedChange" -> Json.toJson(ChangeReturnPeriod)(RequestedChanges.DESApi1365Writes(mockAppConfig)),
+    "returnPeriods" -> Json.toJson(updatedReturnPeriod),
+    "declaration" -> DeclarationTestConstants.declarationDESJsonlNonAgent
+  )
+
   "UpdateVatSubscription" when {
 
     "serializing to JSON" should {
 
       "for the latest DES API1365 writes" should {
 
         "Output the correct JSON for UpdateVatSubscriptionModelMax" in {
-          UpdateVatSubscription.DESApi1365Writes.writes(updateVatSubscriptionModelMax) shouldBe
+          UpdateVatSubscription.DESApi1365Writes(mockAppConfig).writes(updateVatSubscriptionModelMax) shouldBe
```

Comment:
Similar to above, once the JSON is updated then the model here `updateVatSubscriptionModelMax` will also need to be updated so that `ContactPreference` is not `None`